diff --git a/node_modules/@blocknote/react/types/src/schema/ReactBlockSpec.d.ts b/node_modules/@blocknote/react/types/src/schema/ReactBlockSpec.d.ts
index 531e56c..0f18d4d 100644
--- a/node_modules/@blocknote/react/types/src/schema/ReactBlockSpec.d.ts
+++ b/node_modules/@blocknote/react/types/src/schema/ReactBlockSpec.d.ts
@@ -20,7 +20,7 @@ export declare function BlockContentWrapper<BType extends string, PSchema extend
     domAttributes?: Record<string, string>;
     children: ReactNode;
 }): import("react/jsx-runtime").JSX.Element;
-export declare function createReactBlockSpec<const T extends CustomBlockConfig, const I extends InlineContentSchema, const S extends StyleSchema>(blockConfig: T, blockImplementation: ReactCustomBlockImplementation<T, I, S>): {
+export declare function createReactBlockSpec<T extends CustomBlockConfig, I extends InlineContentSchema, S extends StyleSchema>(blockConfig: T, blockImplementation: ReactCustomBlockImplementation<T, I, S>): {
     config: T;
     implementation: import("@blocknote/core").TiptapBlockImplementation<T, any, InlineContentSchema, StyleSchema>;
 };
